api/client-server/registration.yaml

```
@@ -343,6 +342,11 @@ paths:
                 type: string
                 description: The new password for the account.
                 example: "ihatebananas"
+              logout_devices:
+                type: boolean
+                description: |-
+                  Whether other access tokens should be revoked if the request succeeds. Defaults to true.
```

The MSC says it only affects the user's access tokens, and that it affects devices too:
```suggestion
                  Whether the other access tokens, and their associated devices, for the user should be 
                  revoked if the request succeeds. Defaults to true.
```